### PR TITLE
add task updates

### PR DESCRIPTION
**Pull request by meganrogge**
not sure which are worthy of an image

### CODE REVIEW

**Review by Tyriar — APPROVED** [review]

File: release-notes/v1_69.md [hunk 1]
```diff
@@ -200,9 +200,39 @@ When the API to show this hint is finalized, other debuggers will be able to sho
 There is now a Quick Access menu to view a list of your debug consoles! In the menu, you can select a console name to show it in the bottom panel. Here, you can search and filter through your console names.
 
 ![filtering and selecting a debug console from the quick access](images/1_69/debug-filter-and-select.gif)
-
 _Theme: [GitHub Dark Dimmed](https://marketplace.visualstudio.com/items?itemName=GitHub.github-vscode-theme)_
```
Comment (meganrogge) [inline release-notes/v1_69.md:203]:
```suggestion

_Theme: [GitHub Dark Dimmed](https://marketplace.visualstudio.com/items?itemName=GitHub.github-vscode-theme)_
```

File: release-notes/v1_69.md [hunk 2]
```diff
@@ -203,6 +203,37 @@ There is now a Quick Access menu to view a list of your debug consoles! In the m
...
+    "command": "echo test",
+    "icon": { "id": "light-bulb", "color": "terminal.ansiBlue"}
+},
+
+```
+![The task icon appears in the terminal tabs list](./images/1_69/task-icon.png)
+
+Test tasks have the `beaker` icon by default.
+
 There are two ways to access this Quick Access menu:
```
Comment (Tyriar) [inline release-notes/v1_69.md:238]: Bad merge in the middle of debug content

**Review by Tyriar — CHANGES_REQUESTED** [review]

File: release-notes/v1_69.md (hunk 2 above)
Comment (meganrogge) [reply release-notes/v1_69.md:238]: yep fixing

File: release-notes/v1_69.md [hunk 3]
```diff
@@ -295,10 +304,12 @@ When there are too many folding ranges, document symbols or color decorators to
 
 <!-- TODO: mjbvz -->
 
+>>>>>>> vnext
```
Comment (meganrogge) [inline release-notes/v1_69.md:307]:
```suggestion
```

**Review by Tyriar — APPROVED** [review]

**Pull request merged**